Packs/CVE_2021_44228/README.md

```diff
@@ -0,0 +1,28 @@
+This pack is part of the Rapid Breach Response pack.
+
+**Critical RCE Vulnerability: log4j - CVE-2021-44228** refers to a 0-day exploit in the popular Java logging library log4j2.
+
+On Dec. 9, 2021, a remote code execution (RCE) vulnerability in Apache log4j 2 was identified being exploited in the wild. Public proof of concept (PoC) code was released and subsequent investigation revealed that exploitation was incredibly easy to perform.
+
+**Affected Version**
+
+Apache Log4j 2.x <= 2.15.0-rc1
```

Comment:
The arrow should be in the opposite direction? @melamedbn

Reply:
Nope, seems good. also present in Unit42 blog.